.github/workflows/roll_torch_mlir.yml

```diff
@@ -0,0 +1,60 @@
+name: Roll PyTorch
+
+on:
+  schedule:
+    - cron: '0 13 * * *'
+  workflow_dispatch:
+
+jobs:
+  build_linux:
+    name: roll torch-mlir
+    needs: setup
```

Comment:
There is no `setup` job here?

Reply:
I was pulling this from another yaml, dont think its needed if just using a small github hosted runner

Reply:
We need to remove this line if setup job is not needed; otherwise the workflow will fail